Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -614,22 +630,22 @@ private TOTPCacheData getTOTPExpensive(UUID uuid, JedisPool redisPool, Configura
         // SQL
         try {
             TOTPData result = null;
-            if (sqlType == SQLType.MySQL) {
-                result = MySQL.getTOTPData(uuid, sql, storageConfigNode).get();
-            } else if (sqlType == SQLType.SQLite) {
-                result = SQLite.getTOTPData(uuid, sql, storageConfigNode).get();
+            if (ConfigUtil.getSQLType() == SQLType.MySQL) {
+                result = MySQL.getTOTPData(uuid, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode()).get();
+            } else if (ConfigUtil.getSQLType() == SQLType.SQLite) {
+                result = SQLite.getTOTPData(uuid, ConfigUtil.getSQL(), ConfigUtil.getStorageConfigNode()).get();
             }
 
             if (result != null) {
-                if (debug) {
+                if (ConfigUtil.isDebugging()) {
                     logger.info(uuid + " found in storage. Value: " + result);
                 }
                 // Update messaging/Redis, force same-thread
-                Redis.update(result, redisPool, redisConfigNode).get();
-                RabbitMQ.broadcast(result, rabbitConnection).get();
+                Redis.update(result, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode()).get();
+                RabbitMQ.broadcast(result, getRabbitConnection()).get();
```

Comment:
Not using the broadcast method, here?
Redis & Rabbit broadcast/update should probably be in one method, if we're going to make it a method anyway.